src/MCPServer/lib/server/packages.py

```diff
@@ -55,59 +57,63 @@ def _get_setting(name):
 # the workflow data but in the first iteration we've decided to do it this way.
 # There is also the hope that the watched directories can be deprecated in the
 # near future.
+ACTIVE_TRANSFERS_DIR = "activeTransfers"
+WATCHED_ACTIVE_TRANSFERS = os.path.join(
+    _get_setting("WATCH_DIRECTORY"), ACTIVE_TRANSFERS_DIR
+)
 PACKAGE_TYPE_STARTING_POINTS = {
-    "standard": StartingPoint(
+    amtypes.TRANSFER_TYPE_STANDARD: StartingPoint(
         watched_dir=os.path.join(
-            _get_setting("WATCH_DIRECTORY"), "activeTransfers/standardTransfer"
+            WATCHED_ACTIVE_TRANSFERS, amtypes.WATCHED_DIRECTORY_STANDARD
```

Comment:
More transfer-related properties? Could be associated with this data in a new module describing transfer types. (For the present).

Reply:
Why do we also change the watched directories in this PR though when we want to keep scope tight?

1. We can related these directly to the transfer type. The lookup here is based on them. That goes back to point 1 where a future state might describe these in a separate module, but we're not in the habit of moving swathes of code in one go.
2. The use of this on the left `"activeTransfers/standardTransfer"` is not system agnostic, and has long been our practice to try and use `os.path.join(...)` wherever possible. It wasn't caught during PR previously, but there's an opportunity to make that change here.